WirelessUtils.cs

```diff
@@ -32,9 +32,17 @@ public static bool IsTransmitter(Point16 point)
 				return false;
 			
 			Tile tile = Main.tile[point.X, point.Y];
-			if(tile.IsActive && tile.frameY == 18)
-				return (tile.type == ModContent.TileType<Tiles.WirelessTransmitter>()) || (tile.type == ModContent.TileType<Tiles.WirelessTransceiver>());
+			if(tile.HasTile && tile.TileFrameY == 18)
+				return (tile.TileType == ModContent.TileType<Tiles.WirelessTransmitter>()) || (tile.TileType == ModContent.TileType<Tiles.WirelessTransceiver>());
 			return false;
 		}
+
+		public static bool AlreadyExists(Point16 key, Point16 value)
```

Comment:
I believe Wireless technically allows many transmitters to 1 receiver. So instead of ContainsValue(value) you should check for Links[key] == value

Reply:
This whole method may be unnecessary but I may be wrong

Reply:
If I remember right it is to prevent a bug where you remove only one link even though you could add two links so your left with a ghost link.

Reply:
I still don't understand why you check for `ContainsValue` when we agree that the Links map can contain multiples of the same receiver

Reply:
I now see what you mean I wanted to find out if the dictionary already contained that key pair, but I was checking them independently from each other.